= How to use an Oxford Nanopore MinION to extract DNA from river water and determine which bacteria live in it

This article gives an idea of how this kind of biological experiment feels like to a software engineer who has never done any link:/#biology[biology] like link:/#ciro-santilli[Ciro]. Parent node: link:/#articles[].

toc::[]

== Setting

https://www.puntseq.co.uk/[PuntSeq] is a side project led by a few https://en.wikipedia.org/wiki/University_of_Cambridge[Cambridge University] PhDs that aims to determine which bacteria are present in the https://en.wikipedia.org/wiki/River_Cam[River Cam].

In July 2019, the PuntSeq team got together with the awesome https://biomake.space[Cambridge Biomakespace], an awesome biology makerspace open to all, to create a two day science outreach activity showing their procedures.

Ciro knows nothing about biology, but since he is link:/#molecular-biology-is-the-next-big-thing-so-do-anything-in-that-area[very curious about it], in particular he jumped at this opportunity, and decided to document things as well as his limited knowledge would allow.

All participants chipped in some money to help cover the experiment's costs. Ciro suspects that this activity was done partially to help crowdfund the experiment, but it was a worthy investment!

The impressions you get from the experiment as a software engineer will be:

* OMG, this is so labour intensive, why haven't they automated this
* OMG, this is frightening, all the 8 hours of work I've just done are present in that tiny plastic tube
* Amazing! Look at that apparatus! And the bio people are like: I've used this a million times, it's cheap and every lab has one, just work faster and don't break you piece of junk!

== Overview of the experiment

For those that know biology and just want to do the thing, see: xref:protocols-used[xrefstyle=full].

The PuntSeq team uses an xref:oxford-nanopore-minion[] sequencer made by https://en.wikipedia.org/wiki/Oxford_Nanopore_Technologies[Oxford Nanopore Technologies] to sequence the https://en.wikipedia.org/wiki/16S_ribosomal_RNA[16S] region of bacterial DNA, which is about 1500 nucleotides long.

The 16S region codes for one of the RNA pieces that makes the https://en.wikipedia.org/w/index.php?title=Ribosome&oldid=912600990#Bacterial_ribosomes[bacterial ribosome]. https://en.wikipedia.org/wiki/Eukaryote[Eukaryotes] don't have an analogous ribosome part.

Before sequencing the DNA, we will do a https://en.wikipedia.org/wiki/Polymerase_chain_reaction[PCR] with primers that fit just before and just after the 16S DNA, in well conserved regions expected to be present in all bacteria.

This way, we amplify only the 16S region of bacteria, excluding other parts of bacterial genome, and excluding prokaryotes entirely.

Despite coding such a fundamental piece of RNA, there is still surprisingly variability in the 16S region across different bacteria, and it is those differences will allow us to identify which bacteria are present in the river.

The variability exists because certain base pairs are not fundamental for the function of the 16S region. This variability happens mostly on https://en.wikipedia.org/wiki/Stem-loop[RNA loops as opposed to stems], i.e. parts of the RNA that don't base pair with other RNA in the https://en.wikipedia.org/wiki/Nucleic_acid_secondary_structure[RNA secondary structure] as shown at: xref:listing-rna-stem-loop[xrefstyle=full].

[[listing-rna-stem-loop]]
.RNA stem-loop structure
----
                A-U
               /   \
A-U-C-G-A-U-C-G     C
| | | | | | | |     |
U-A-G-C-U-A-G-C     G
               \   /
                U-A
|             ||    |
+-------------++----+
    stem        loop
----

This is how the 16S RNA secondary structure looks like in its full glory: xref:image-16s-rna-secondary-structure[xrefstyle=full].

.16S RNA secondary structure
wikimedia_image::https://upload.wikimedia.org/wikipedia/commons/a/a6/16S.svg[height=800]

Since loops don't base pair, they are less crucial in the determination of the secondary structure of the RNA.

The variability may however not be large enough to allow us to differentiate between related species which might have the same sequence, but we should at least be able to obtain at least a reasonable https://en.wikipedia.org/wiki/Phylum[phylum] breakdown.

=== Why Oxford Nanopore was used instead of Illumina for the sequencing

link:++https://en.wikipedia.org/wiki/Illumina,_Inc.++[Illumina equipment] is currently cheaper per base pair and dominates the human genome sequencing market, but it requires a much higher initial investment for the equipment

The Nanopore costs just about 500 pounds for the reusable machine, and 500 pounds for the one usage flow cell which can decode up to 30 billion base pairs, which is about 10 human genomes!

Other advantages of the MinION over Illumina which didn't really matter to this particular experiment are:

* portability for e.g. to do analysis on the field near infections outbreaks
* long reads which can be necessary for long repetitive regions

One disadvantage of the Oxford Nanopore is that its error rate per base pair read is higher than Illumina.

=== Metagenomics

https://en.wikipedia.org/wiki/Metagenomics

This kind of "sequence some random DNA to determine what is present in the sample" analysis is called metagenomics.

Besides river sample, other types of samples where of metagenomics is used include:

* food, including searching for desirable microorganisms such as in cheese or bread yeast
* poo, e.g. to study how the human microbiome influences health. There are companies actively working on this, e.g.: https://www.microbiotica.com/
* blood, e.g. to search for https://en.wikipedia.org/wiki/Circulating_tumor_DNA[Circulating tumor DNA]

== Sample collection

As you would expect, not much secret here, we just dumped a 1L glass bottle with a rope attached around the neck in the river, and pulled it out with the rope.

The temperature of the water was measured with a mercury thermometer, and the PH with pH strips and a cell phone app to compare the color of the strip.

There were some swans in the river, so... swan poo bacteria I guess?

And, in the name of science, we even wore gloves to not contaminate the samples!

Who said you can't have fun with science? xref:video-river-water-sample-collection-with-a-bottle-and-string[xrefstyle=full].

wikimedia_image::https://upload.wikimedia.org/wikipedia/commons/thumb/3/33/River_water_sample_collection_swans.jpg/800px-River_water_sample_collection_swans.jpg[]

wikimedia_image::https://upload.wikimedia.org/wikipedia/commons/thumb/a/a9/River_water_sample_collection_tie_rope_to_bottle.jpg/360px-River_water_sample_collection_tie_rope_to_bottle.jpg[]

wikimedia_image::https://upload.wikimedia.org/wikipedia/commons/thumb/9/9b/River_water_sample_collection_get_sample.jpg/360px-River_water_sample_collection_get_sample.jpg[]

wikimedia_image::https://upload.wikimedia.org/wikipedia/commons/thumb/7/75/River_water_sample_collection_measure_temperature.jpg/360px-River_water_sample_collection_measure_temperature.jpg[]

wikimedia_image::https://upload.wikimedia.org/wikipedia/commons/thumb/4/4f/River_water_sample_collection_read_PH_strip.jpg/360px-River_water_sample_collection_read_PH_strip.jpg[]

wikimedia_image::https://upload.wikimedia.org/wikipedia/commons/thumb/0/0a/River_water_sample_collection_identify_bottle.jpg/360px-River_water_sample_collection_identify_bottle.jpg[]

wikimedia_video::https://upload.wikimedia.org/wikipedia/commons/transcoded/b/bb/River_water_sample_collection_with_a_bottle_and_string.ogv/River_water_sample_collection_with_a_bottle_and_string.ogv.480p.vp9.webm[]

== DNA extraction

The first thing we had to do with the sample was to extract the DNA present in the water in a pure form for the PCR.

We did that with a xref:qiagen-dneasy-powerwater-kit[].

As you would expect, this consists of a purification procedure with several steps.

In each step we take a physical or chemical action on the sample, which splits it into two parts: the one with the DNA and the one without.

We then take the part with the DNA, and throw away the one without the DNA.

The first steps are coarser, and finer and finer splits are done as we move forward.

=== Filtration with vacuum pump

The first thing we did was to filter the water samples with a filter that is so fine that not even DNA can pass through, but water can.

This will reduce the 1 liter volumes to more manageable amounts. Reagents are expensive, and centrifuges are small!

Since the filter is so fine, filtering by gravity alone would take forever, and so we used a vacuum pump to speed thing up!

For that we used:

* xref:thermo-scientific-nalgene-polysulfone-reusable-bottle-top-filters[]
* xref:knf-laboport-series-laboratory-vacuum-pump[]

wikimedia_image::https://upload.wikimedia.org/wikipedia/commons/6/6e/Vacuum_pump_filter_peel_filter.png[]

wikimedia_image::https://upload.wikimedia.org/wikipedia/commons/7/78/Vacuum_pump_filter_place_filter.png[]

wikimedia_video::https://upload.wikimedia.org/wikipedia/commons/transcoded/3/3f/Vacuum_pump_filter_pour_sample_and_turn_on.webm/Vacuum_pump_filter_pour_sample_and_turn_on.webm.480p.vp9.webm[]

wikimedia_video::https://upload.wikimedia.org/wikipedia/commons/transcoded/8/85/Vacuum_pump_filter_cut_and_place_in_eppendorf.webm/Vacuum_pump_filter_cut_and_place_in_eppendorf.webm.480p.vp9.webm[]

=== Post filtration purification

Now that we had the DNA in Eppendorfs, we were ready to continue the purification in a simpler and more standardized lab pipeline fashion.

First we added some small specialized stones and chemicals to the water and shook them Eppendorfs hard in a xref:scientific-industries-inc-vortex-genie-2[] machine to break the cells and free the DNA.

wikimedia_video::https://upload.wikimedia.org/wikipedia/commons/transcoded/e/e1/Scientific_Industries_Inc_Vortex-Genie_2_loading.webm/Scientific_Industries_Inc_Vortex-Genie_2_loading.webm.480p.vp9.webm[]

wikimedia_video::https://upload.wikimedia.org/wikipedia/commons/transcoded/1/19/Scientific_Industries_Inc_Vortex-Genie_2_running.ogv/Scientific_Industries_Inc_Vortex-Genie_2_running.ogv.480p.vp9.webm[]

Once that was done, we added several reagents which split the solution into two phases: one containing the DNA and the other not. We would then pipette the phase with the DNA out to the next Eppendorf, and continue the process.

In one step for example, the DNA was present as a white precipitate at the bottom of the tube, and we threw away the supernatant liquid: xref:image-qiagen-dneasy-powerwater-kit-white-precipitate[xrefstyle=full].

wikimedia_image::https://upload.wikimedia.org/wikipedia/commons/thumb/3/30/Qiagen_DNeasy_PowerWater_Kit_White_Precipitate.jpg/586px-Qiagen_DNeasy_PowerWater_Kit_White_Precipitate.jpg[]

At various stages, centrifuging was also necessary. Much like the previous vacuum pump step, this adds extra gravity to speed up the separation of phases with different molecular masses.

In our case, we used a xref:vwr-micro-star-17-microcentrifuge[] for those steps:

wikimedia_image::https://upload.wikimedia.org/wikipedia/commons/thumb/0/03/VWR_Micro_Star_17_microcentrifuge.jpg/360px-VWR_Micro_Star_17_microcentrifuge.jpg[]

wikimedia_image::https://upload.wikimedia.org/wikipedia/commons/thumb/6/65/VWR_Micro_Star_17_microcentrifuge_loading.png/358px-VWR_Micro_Star_17_microcentrifuge_loading.png[]

Then, when we had finally finished all the purification steps, we measured the quantity of DNA with a xref:biochrom-simplinano-spectrophotometer[] to check that the purification went well:

wikimedia_image::https://upload.wikimedia.org/wikipedia/commons/thumb/4/47/Biochrom_SimpliNano_spectrophotometer_loading_sample.jpg/262px-Biochrom_SimpliNano_spectrophotometer_loading_sample.jpg[]

wikimedia_image::https://upload.wikimedia.org/wikipedia/commons/thumb/f/f4/Biochrom_SimpliNano_spectrophotometer_result_readout.jpg/360px-Biochrom_SimpliNano_spectrophotometer_result_readout.jpg[]

And because the readings were good, we put it in our -20 C fridge to preserve it until the second day of the workshop and called it a day:

wikimedia_image::https://upload.wikimedia.org/wikipedia/commons/thumb/f/f7/Minus_20_fridge_storing_samples.jpg/183px-Minus_20_fridge_storing_samples.jpg[]

== PCR

PCR protocols are very standard it seems, all that biologists need to know to reproduce is the time and temperature of each step.

We did 35 cycles of:

* 94˚C for 30 seconds
* 60˚C for 30 seconds
* 72˚C for 45 seconds

Because it is considered the less interesting step, and because it takes quite some time, this step was done by the event organizers between the two event days, so I did not get to take many photos.

This process used a xref:marshal-scientific-mj-research-ptc-200-thermal-cycler[]:

wikimedia_image::https://upload.wikimedia.org/wikipedia/commons/thumb/f/f5/Marshal_Scientific_MJ_Research_PTC-200_Thermal_Cycler.jpg/360px-Marshal_Scientific_MJ_Research_PTC-200_Thermal_Cycler.jpg[]

We added PCR primers for regions that surround the 16S DNA. The primers are just bought from a vendor, and we used well known regions are called 27F and 1492R. Here is a paper that analyzes other choices: https://academic.oup.com/femsle/article/221/2/299/630719 (http://web.archive.org/web/20190911091818/https://academic.oup.com/femsle/article/221/2/299/630719[archive]) "Evaluation of primers and PCR conditions for the analysis of 16S rRNA genes from a natural environment" by "Yuichi Hongoh, Hiroe Yuzawa, Moriya Ohkuma, Toshiaki Kudo Published" published 01 April 2003.

One cool thing about the PCR is that we can also add a known barcode at the end of each primer as shown at xref:listing-pcr-diagram[xrefstyle=full]. This way, we were able to:

* add a different barcode for samples collected from different locations
* sequence them all in one go
* then just from the sequencing output the barcode to determine where each sequence came from!

[[listing-pcr-diagram]]
.PCR diagram
----
Bacterial DNA (a little bit)
... --- 27S --- 16S --- 1492R --- ...

|
|
v

PCR output (a lot of)
Barcode --- 27S --- 16S --- 1492R
----

Finally, after purification, we used the xref:qiagen-qiaquick-pcr-purification-kit[] protocol to purify the generated from unwanted PCR byproducts.

== Sequencing

Once we had the amplified 16S DNA clean, we were almost ready to start sequencing!

First we had to remove PCR byproducts with the xref:qiagen-qiaquick-pcr-purification-kit[].

Then we prepared the DNA for sequencing with the xref:oxford-nanopore-sqk-lsk109-ligation-sequencing-kit[]

After all was said and done, the DNA of the several 1l water bottles and hours of hard purification labour were contained in one single Eppendorf!

Once the sample was fully prepared, we opened the Nanopore flow cell, which is the 500 dollar consumable piece that goes in the sequencer.

We then had to pipette the final golden Eppendorf into the flow cell. My anxiety levels were going through the roof: xref:image-oxford-nanopore-minion-flow-cell-pipette-loading[xrefstyle=full].

wikimedia_image::https://upload.wikimedia.org/wikipedia/commons/thumb/8/81/Oxford_nanopore_MinION_flow_cell_package.jpg/304px-Oxford_nanopore_MinION_flow_cell_package.jpg[]

wikimedia_image::https://upload.wikimedia.org/wikipedia/commons/thumb/0/00/Oxford_nanopore_MinION_flow_cell_front.jpg/640px-Oxford_nanopore_MinION_flow_cell_front.jpg[]

wikimedia_image::https://upload.wikimedia.org/wikipedia/commons/thumb/c/c2/Oxford_nanopore_MinION_flow_cell_back.jpg/1024px-Oxford_nanopore_MinION_flow_cell_back.jpg[]

wikimedia_image::https://upload.wikimedia.org/wikipedia/commons/thumb/f/f8/Oxford_nanopore_MinION_flow_cell_pipette_loading.jpg/278px-Oxford_nanopore_MinION_flow_cell_pipette_loading.jpg[]

At this point bio people start telling lab horror stories of expensive solutions being spilled and people having to recover them from fridge walls, or of how people threw away golden Eppendorfs and had to pick them out of trash bins with hundreds of others looking exactly the same etc. (but also how some discoveries were made like this) This reminded me of: https://youtu.be/89UNPdNtOoE?t=919 https://en.wikipedia.org/wiki/Alfred_Maddock[Alfred Maddock's plutonium horror story].

Luckily this time, it worked out!

We then just had to connect the MinION to the computer, and wait for 2 days.

During this time, the DNA would be sucked through the pores.

As can be seen from xref:video-oxford-nanopore-minion-software-channels-pannel-on-mac[] the software tells us which pores are still working.

wikimedia_image::https://upload.wikimedia.org/wikipedia/commons/thumb/0/03/Oxford_Nanopore_MinION_connected_to_a_Mac_via_USB.jpg/360px-Oxford_Nanopore_MinION_connected_to_a_Mac_via_USB.jpg[]

wikimedia_video::https://upload.wikimedia.org/wikipedia/commons/transcoded/7/7e/Oxford_Nanopore_MinION_software_channels_pannel_on_Mac.webm/Oxford_Nanopore_MinION_software_channels_pannel_on_Mac.webm.480p.webm[]

Pores go bad sooner or later randomly, until there are none left, at which point we can stop the process and throw the flow cell away.

48 hours was expected to be a reasonable time until all pores went bad, and so we called it a day, and waited for an email from the PuntSeq team telling us how things went.

The nominal maximum base pairs sequenced is 30 billion, so about 10 entire human genomes, which is not bad! We reached a yield of 16Gbp which the bio people said was not bad.

== Bioinformatics

To be honest, because I'm a software engineer, and I've done enough staring in computers for a lifetime already, and I believe in the power of https://en.wikipedia.org/wiki/Git[Git], I didn't pay much attention to this ;-)

I can however see that it does however present interesting problems!

Because we had to wait for 2 days to get our data, the workshop first reused sample data from previous collections done earlier in the year.

First there is some signal processing / machine learning required to do the https://en.wikipedia.org/wiki/Base_calling[base calling], which is not trivial in the Oxford Nanopore, since neighbouring bases can affect the signal of each other. This is mostly handled by Oxford Nanopore itself, or by hardcore programmers in the field however.

After the base calling was done, we analyzed the data using computer programs that match the sequenced 16S sequences to a database of known sequenced species.

This is of course not just a simple direct string matching problem, since like any in experiment, the DNA reads have some errors, so the program has to find the best match even though it is not exact.

The PuntSeq team uploaded / will upload the data to well known open databases so that it will be preserved forever! TODO link.

== Conclusions

* against all odds, the experiment worked and we got DNA out of the water, despite a bunch of non-bio newbs actively messing with random parts of the experiment
* PuntSeq and Biomakespace people, and all those tho do scientific outreach, are awesome!
* biology is hard
* creating insanely media rich articles like this is also hard, but the following helped enormously:
** link:/#media-rationale-of-ciro-santillis-website[Wikimedia Commons] to store large media files out of Git
** link:/markdown-style-guide[Asciidoctor] extensions to easily include those media files
** https://unix.stackexchange.com/questions/25978/image-viewer-for-multiple-images/539333#539333[Nomacs] to give Google Photos photos meaningful names and to edit people's faces out of pictures ;-)
* some scientific Wikipedia pages may or may not have been edited with better pictures during the course of writing this article

== Protocols used

Protocols are the biologist term for "recipe".

I found that a lot of biology comes down to this: get the right recipe, follow it well even though you don't understand all the proprietary details, and pray.

=== Qiagen DNeasy PowerWater Kit

https://www.qiagen.com/gb/products/discovery-and-translational-research/dna-rna-purification/dna-purification/microbial-dna/dneasy-powerwater-kit (http://web.archive.org/web/20190905084344/https://www.qiagen.com/gb/products/discovery-and-translational-research/dna-rna-purification/dna-purification/microbial-dna/dneasy-powerwater-kit/[archive]) Here is its documentation: https://www.qiagen.com/gb/resources/download.aspx?id=bb731482-874b-4241-8cf4-c15054e3a4bf&lang=en (http://web.archive.org/web/20190905084623/https://www.qiagen.com/gb/resources/download.aspx?id=bb731482-874b-4241-8cf4-c15054e3a4bf&lang=en[archive]).

Manual archive: http://web.archive.org/web/20190911111136/https://www.qiagen.com/gb/resources/download.aspx?id=bb731482-874b-4241-8cf4-c15054e3a4bf&lang=en

Kit to extract clean DNA from water.

=== Qiagen QIAquick PCR Purification Kit

https://www.qiagen.com/us/products/discovery-translational-research/dna-rn-a-purification/dna-purification/dna-clean-up/qiaquick-pcr-purification-kit/#orderinginformation (http://web.archive.org/web/20190911092647/https://www.qiagen.com/us/products/discovery-translational-research/dna-rn-a-purification/dna-purification/dna-clean-up/qiaquick-pcr-purification-kit/[archive])

Manual archive: http://web.archive.org/web/20190911100243/https://www.qiagen.com/us/resources/download.aspx?id=e0fab087-ea52-4c16-b79f-c224bf760c39&lang=en

Removes PCR byproducts from purified DNA.

=== Oxford Nanopore SQK-LSK109 Ligation Sequencing Kit

https://store.nanoporetech.com/ligation-sequencing-kit.html (http://web.archive.org/web/20190911092756/https://store.nanoporetech.com/ligation-sequencing-kit.html[archive])

Repairs the ends of DNA, and also attaches an adapter protein to the DNA that makes them go through the pores of e.g. an xref:oxford-nanopore-minion[].

== Equipment used

=== Oxford Nanopore MinION

https://nanoporetech.com/products/minion (https://web.archive.org/web/20190825022606/https://nanoporetech.com/products/minion[archive])

.Oxford Nanopore MinION top
wikimedia_image::https://upload.wikimedia.org/wikipedia/commons/thumb/5/57/Oxford_Nanopore_MinION_top_cropped.jpg/392px-Oxford_Nanopore_MinION_top_cropped.jpg[]

.Oxford Nanopore MinION side
wikimedia_image::https://upload.wikimedia.org/wikipedia/commons/thumb/6/6e/Oxford_Nanopore_MinION_side_cropped.jpg/191px-Oxford_Nanopore_MinION_side_cropped.jpg[]

.Oxford Nanopore MinION top open
wikimedia_image::https://upload.wikimedia.org/wikipedia/commons/thumb/0/0a/Oxford_Nanopore_MinION_top_open_cropped.jpg/110px-Oxford_Nanopore_MinION_top_open_cropped.jpg[]

.Oxford Nanopore MinION side USB
wikimedia_image::https://upload.wikimedia.org/wikipedia/commons/thumb/0/0f/Oxford_Nanopore_MinION_side_USB_cropped.jpg/597px-Oxford_Nanopore_MinION_side_USB_cropped.jpg[]

wikimedia_image::https://upload.wikimedia.org/wikipedia/commons/thumb/8/81/Oxford_nanopore_MinION_flow_cell_package.jpg/304px-Oxford_nanopore_MinION_flow_cell_package.jpg[]

wikimedia_image::https://upload.wikimedia.org/wikipedia/commons/thumb/0/00/Oxford_nanopore_MinION_flow_cell_front.jpg/640px-Oxford_nanopore_MinION_flow_cell_front.jpg[]

wikimedia_image::https://upload.wikimedia.org/wikipedia/commons/thumb/c/c2/Oxford_nanopore_MinION_flow_cell_back.jpg/1024px-Oxford_nanopore_MinION_flow_cell_back.jpg[]

wikimedia_image::https://upload.wikimedia.org/wikipedia/commons/thumb/f/f8/Oxford_nanopore_MinION_flow_cell_pipette_loading.jpg/278px-Oxford_nanopore_MinION_flow_cell_pipette_loading.jpg[]

wikimedia_image::https://upload.wikimedia.org/wikipedia/commons/thumb/0/03/Oxford_Nanopore_MinION_connected_to_a_Mac_via_USB.jpg/360px-Oxford_Nanopore_MinION_connected_to_a_Mac_via_USB.jpg[]

wikimedia_video::https://upload.wikimedia.org/wikipedia/commons/transcoded/7/7e/Oxford_Nanopore_MinION_software_channels_pannel_on_Mac.webm/Oxford_Nanopore_MinION_software_channels_pannel_on_Mac.webm.480p.webm[]

=== Thermo Scientific Nalgene Polysulfone Reusable Bottle Top Filters

https://www.fishersci.no/shop/products/nalgene-polysulfone-reusable-bottle%20-top-filters/10465781 (http://web.archive.org/web/20190907131756/https://www.fishersci.no/shop/products/nalgene-polysulfone-reusable-bottle%20-top-filters/10465781[archive])

This is where we poured the water. It was not large enough for the entire 1L sample, so we had to do it a few times.

=== KNF Laboport series laboratory vacuum pump

https://www.knfusa.com/en/laboport/ (http://web.archive.org/web/20190907132036/https://www.knfusa.com/en/laboport/[archive]).

wikimedia_image::https://upload.wikimedia.org/wikipedia/commons/thumb/5/52/KNF_Laboport_series_laboratory_vacuum_pump.jpg/640px-KNF_Laboport_series_laboratory_vacuum_pump.jpg[]

=== Scientific Industries Inc. Vortex-Genie 2

https://www.scientificindustries.com/vortex-genie-2.html (http://web.archive.org/web/20190908034549/https://www.scientificind.ustries.com/vortex-genie-2.html[archive])

wikimedia_video::https://upload.wikimedia.org/wikipedia/commons/e/e1/Scientific_Industries_Inc_Vortex-Genie_2_loading.webm[]

wikimedia_video::https://upload.wikimedia.org/wikipedia/commons/1/19/Scientific_Industries_Inc_Vortex-Genie_2_running.ogv[]

=== VWR Micro Star 17 microcentrifuge

https://uk.vwr.com/store/product/8306728/microcentrifuges-ventilated-refrigerated-micro-star-17-17r (http://web.archive.org/web/20190908040119/https://uk.vwr.com/store/product/8306728/microcentrifuges-ventilated-refrigerated-micro-star-17-17r[archive]).

wikimedia_image::https://upload.wikimedia.org/wikipedia/commons/thumb/6/65/VWR_Micro_Star_17_microcentrifuge_loading.png/358px-VWR_Micro_Star_17_microcentrifuge_loading.png[]

wikimedia_image::https://upload.wikimedia.org/wikipedia/commons/thumb/0/03/VWR_Micro_Star_17_microcentrifuge.jpg/360px-VWR_Micro_Star_17_microcentrifuge.jpg[]

=== VELP Scientifica WIZARD IR Infrared Vortex Mixer

https://www.velp.com/en/products/lines/3/family/44/vortex_mixers/65/wizard_ir_infrared_vortex_mixer (http://web.archive.org/web/20190908091343/https://www.velp.com/en/products/lines/3/family/44/vortex_mixers/65/wizard_ir_infrared_vortex_mixer[archive]).

wikimedia_image::https://upload.wikimedia.org/wikipedia/commons/thumb/5/5b/VELP_Scientifica_WIZARD_IR_Infrared_Vortex_Mixer_running.jpg/360px-VELP_Scientifica_WIZARD_IR_Infrared_Vortex_Mixer_running.jpg[]

=== Marshal Scientific MJ Research PTC-200 Thermal Cycler

https://www.marshallscientific.com/MJ-Research-PTC-200-Thermal-Cycler-p/mj-200.htm (http://web.archive.org/web/20190908091629/https://www.marshallscientific.com/MJ-Research-PTC-200-Thermal-Cycler-p/mj-200.htm[archive]).

wikimedia_image::https://upload.wikimedia.org/wikipedia/commons/thumb/f/f5/Marshal_Scientific_MJ_Research_PTC-200_Thermal_Cycler.jpg/360px-Marshal_Scientific_MJ_Research_PTC-200_Thermal_Cycler.jpg[]

=== GE MagRack 6

https://www.gelifesciences.com/en/us/shop/protein-analysis/protein-sample-preparation/protein-enrichment/magrack-6-p-05761 (http://web.archive.org/web/20190908091852/https://www.gelifesciences.com/en/us/shop/protein-analysis/protein-sample-preparation/protein-enrichment/magrack-6-p-05761[archive]).

wikimedia_image::https://upload.wikimedia.org/wikipedia/commons/thumb/0/06/GE_MagRack_6_pipetting.jpg/360px-GE_MagRack_6_pipetting.jpg[]

=== BTLab Systems Mini Centrifuge

https://www.btlabsystems.com/Centrifuges/Mini_Centrifuge_Fixed_7K (http://web.archive.org/web/20190908094324/https://www.btlabsystems.com/Centrifuges/Mini_Centrifuge_Fixed_7K[archive]).

Manual: http://web.archive.org/web/20190908094334/https://www.btlabsystems.com/downloads/BT602_Mini_Centrifuge_7K_Fixed.pdf

wikimedia_image::https://upload.wikimedia.org/wikipedia/commons/thumb/c/c2/BTLab_Systems_Mini_Centrifuge_open.jpg/360px-BTLab_Systems_Mini_Centrifuge_open.jpg[]

=== Fischer Scientific UVP LM-26E Benchtop 2UV Transilluminator

https://www.bidspotter.com/en-us/auction-catalogues/bscsur/catalogue-id-bscsur10011/lot-c6605b41-1a14-40e5-a255-a5c5000866e0 (http://web.archive.org/web/20190908094721/https://www.bidspotter.com/en-us/auction-catalogues/bscsur/catalogue-id-bscsur10011/lot-c6605b41-1a14-40e5-a255-a5c5000866e0[archive]) Cannot exact same product on official website, but here is a similar one:  https://www.fishersci.co.uk/shop/products/lm-26-2uv-transilluminator/12382038 (http://web.archive.org/web/20190908094903/https://www.fishersci.co.uk/shop/products/lm-26-2uv-transilluminator/12382038[archive]).

wikimedia_image::https://upload.wikimedia.org/wikipedia/commons/thumb/0/06/Fischer_Scientific_UVP_LM-26E_Benchtop_2UV_Transilluminator.jpg/640px-Fischer_Scientific_UVP_LM-26E_Benchtop_2UV_Transilluminator.jpg[]

=== Biochrom SimpliNano spectrophotometer

https://biochromspectros.com/spectrophotometers/simplinano-cat/simplinano-spectrophotometer.html (http://web.archive.org/web/20190920214435/https://biochromspectros.com/spectrophotometers/simplinano-cat/simplinano-spectrophotometer.html[archive])

Manual: https://biochromspectros.com/media/wysiwyg/support_page/support-simplinano/Simplinano-UM.pdf (https://web.archive.org/web/20190920214755/https://biochromspectros.com/media/wysiwyg/support_page/support-simplinano/Simplinano-UM.pdf[archive])

wikimedia_image::https://upload.wikimedia.org/wikipedia/commons/thumb/4/47/Biochrom_SimpliNano_spectrophotometer_loading_sample.jpg/262px-Biochrom_SimpliNano_spectrophotometer_loading_sample.jpg[]

wikimedia_image::https://upload.wikimedia.org/wikipedia/commons/thumb/f/f4/Biochrom_SimpliNano_spectrophotometer_result_readout.jpg/360px-Biochrom_SimpliNano_spectrophotometer_result_readout.jpg[]
